server/middleware/validation.js

```diff
@@ -46,7 +46,26 @@ const logIn = (req, res, next) => {
     });
 }
 
+const comment = (req, res, next) => {
+    const validationRule = {
+        "comment": "required|string|max:180"
+    }
+    validator(req.body, validationRule, {}, (err, status) => {
+        if (!status) {
+            res.status(412)
```

Comment:
please appropriate error code => 400 and .json

Reply:
ok sir. i have done that